src/cascadia/TerminalSettingsModel/CascadiaSettings.cpp

```diff
@@ -1087,7 +1087,21 @@ bool CascadiaSettings::IsDefaultTerminalAvailable() noexcept
     DWORDLONG dwlConditionMask = 0;
     VER_SET_CONDITION(dwlConditionMask, VER_BUILDNUMBER, VER_GREATER_EQUAL);
 
-    return VerifyVersionInfoW(&osver, VER_BUILDNUMBER, dwlConditionMask) != FALSE;
+    if (VerifyVersionInfoW(&osver, VER_BUILDNUMBER, dwlConditionMask) != FALSE)
+    {
+        return true;
+    }
+
+    static bool isOtherwiseAvailable = [] {
+        wil::unique_hkey key;
+        const auto lResult = RegOpenKeyExW(HKEY_LOCAL_MACHINE,

```

Comment:
If you add a line break in front of `HKEY_LOCAL_MACHINE` the indentation will be a bit neater (or rather, more narrow).

BTW I just realized that we haven't updated clang-format in a while - it's still on v13, but VS already ships with v15! This means we can use `AlignAfterOpenBracket: BlockIndent` now!!